client_test.go

```diff
@@ -897,7 +897,7 @@ func Test_Client_InsertMany(t *testing.T) {
 
 		jobs, err := bundle.queries.JobGetByKind(ctx, client.driver.GetDBPool(), (noOpArgs{}).Kind())
 		require.NoError(t, err)
-		require.Len(t, jobs, 2, fmt.Sprintf("Expected to find exactly two jobs of kind: %s", (noOpArgs{}).Kind()))
+		require.Len(t, jobs, 2, "Expected to find exactly two jobs of kind: "+(noOpArgs{}).Kind()) //nolint:goconst
```

Comment:
Added nolint on these lines ... I dunno about adding constants for assertion messages.